src/components/o-modal/_preview.html

```diff
@@ -0,0 +1,87 @@
+<!--Please create here a HTML example by using just default HTML tags-->
+
+<button onclick="openModal()">Open modal</button>
+
+<axa-modal class="axa-modal-preview">
+  <h2>Prämie berechnen</h2>
+  <p><strong>Mehr als 80%</strong> unserer Kunden kombinieren die Hausratversicherung mit einer Privathaftpflichtversicherung.</p>
+  <p>Was möchten Sie?</p>
+
+  <section class="grid">
+    <div>
+      <svg class="huge">
+        <use xlink:href="#src--assets--icons--rocking_chair" href="#src--assets--icons--rocking_chair" />
+      </svg>
+      <svg class="huge">
+        <use xlink:href="#src--assets--icons--broken_glass" href="#src--assets--icons--broken_glass" />
+      </svg>
+    </div>
+    <div>
+      <svg class="huge">
+        <use xlink:href="#src--assets--icons--rocking_chair" href="#src--assets--icons--rocking_chair" />
```

Comment:
please use `<axa-icon>`

Reply:
I tried but I cannot resize it without blackbox-styling it.

Reply:
should be able to pass a class down using `classes` attribute

Reply:
What the fuck... you are right.